ios/AppLovinMAXAdView.mm

```diff
@@ -119,6 +132,392 @@ + (void)destroyNativeUIComponentAdView:(NSNumber *)adViewId
     resolve(nil);
 }
 
+#ifdef RCT_NEW_ARCH_ENABLED
+
++ (ComponentDescriptorProvider)componentDescriptorProvider
+{
+    return concreteComponentDescriptorProvider<AppLovinMAXAdViewComponentDescriptor>();
+}
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    
+    if ( self )
+    {
+        static const auto defaultProps = std::make_shared<const AppLovinMAXAdViewProps>();
+        _props = defaultProps;
+        
+        const auto &initProps = *std::static_pointer_cast<AppLovinMAXAdViewProps const>(_props);
+        _adaptiveBannerEnabled = initProps.adaptiveBannerEnabled;
+        _autoRefresh = initProps.autoRefresh;
+        _loadOnMount = initProps.loadOnMount;
+        
+        [self setupEventHandlers];
+    }
+    
+    return self;
+}
+
+- (void)setupEventHandlers
+{
+    self.onAdLoadedEvent = [self](NSDictionary *event)
+    {
+        if ( _eventEmitter )
+        {
+            auto adViewEventEmitter = std::static_pointer_cast<AppLovinMAXAdViewEventEmitter const>(_eventEmitter);
+            
+            NSDictionary *size = [event valueForKey: @"size"];
+            
+            AppLovinMAXAdViewEventEmitter::OnAdLoadedEvent result =
+            {
+                .adUnitId = std::string([[event valueForKey: @"adUnitId"] ?: @"" UTF8String]),
```

Comment:
Use subscript notation, and dot notation for `UTF8String` property.

Reply:
The value is type of id, so dot notation can't be used.